notify/impl.go

```diff
@@ -135,6 +136,8 @@ func BuildReceiverIntegrations(nc *config.Receiver, tmpl *template.Template) []I
 
 const contentTypeJSON = "application/json"
 
+var userAgentHeader = fmt.Sprintf("Alertmanager/%s", version.Version)
```

Comment:
Tried to read up on things, but since const is checked at compile time, but it seems like version.Version is filled in by the linker, it may not be possible to have userAgentHeader as a const (and in the linked PR for Prometheus it's a var there as well)